Items/08_Tool_Sets.txt

```diff
@@ -0,0 +1,215 @@
+==============================
+Tool Sets
+==============================
+
+Blade Maintenance and Care Kit ($100):
+	A good whetstone, polish, oil and cleaning supplies are worth
+their weight in gold. These tools keep blades sharp, rust-free and
+ready for action.
+
+Relevant Skill(s): Weapons - Melee, Weapons - Thrown
+Context: Bladed Weapons
+Noise Level: Low
+Weight: 
```

Comment:
Let's remove unknown weights and add them back in when we write encumbrance rules?